src/components/popover-container/popover-container.component.tsx

```diff
@@ -346,13 +343,11 @@ export const PopoverContainer = ({
       <div ref={popoverReference}>
         {renderOpenComponent(renderOpenComponentProps)}
       </div>
-      <Transition
+      <CSSTransition
```

Comment:
The choice to use `CSSTransition` over `Transition` was made to ensure that the styling changes are detectable in the RTL tests.